environments/fetcher/cmd/build.sh

```diff
@@ -1,2 +1,5 @@
 #!/bin/sh
-CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -gcflags=-trimpath=$GOPATH -asmflags=-trimpath=$GOPATH -o fetcher .
+version=$1
+date=$2
+gitcommit=$3
```

Comment:
I am not sure about this: do you always need to specify these values? If so, maybe add some of default values here. Otherwise, it becomes pretty annoying to compile locally. Maybe something like: (date: now(), gitcommit: git rev-parse HEAD, version: $gitcommit)